app/models/hostgroup.rb

```diff
@@ -19,7 +19,7 @@ class Hostgroup < ActiveRecord::Base
   validates :name, :presence => true
   validates :root_pass, :allow_blank => true, :length => {:minimum => 8, :message => _('should be 8 characters or more')}
   has_many :group_parameters, :dependent => :destroy, :foreign_key => :reference_id, :inverse_of => :hostgroup
-  accepts_nested_attributes_for :group_parameters, :allow_destroy => true
+  accepts_nested_attributes_for :group_parameters, :allow_destroy => true, :reject_if => ->(param) { param[:value].empty? }
```

Comment:
Is this needed now?  Using the override button instead of changing the value in the existing parent parameter value box means that the _destroy nested magic should work.  It should allow you to override a parent parameter and set an empty value, separately to destroying it.